man/chpl.txt

```diff
@@ -587,9 +587,6 @@ ENVIRONMENT VARIABLES FOR OPTIONS
   non-empty value of the environment variable equivalent has the same effect
   as passing that option once.
 
-
-ENVIRONMENT
-
   See $CHPL_HOME/doc/chplenv.rst for detailed information about
```

Comment:
Seems like this paragraph should go away now since there aren't really "general environment variable settings" anymore -- they're all linked to flags.  If we need to point to chplenv.rst (and we may not need to), we could do so in the flag descriptions themselves or in the "Compiler Configuration Options" section?